packages/grpc-js/src/subchannel.ts

```diff
@@ -405,14 +411,14 @@ export class Subchannel {
             errorCode === http2.constants.NGHTTP2_ENHANCE_YOUR_CALM &&
             opaqueData.equals(tooManyPingsData)
           ) {
-            logging.log(
-              LogVerbosity.ERROR,
-              `Connection to ${this.channelTarget} rejected by server because of excess pings`
-            );
             this.keepaliveTimeMs = Math.min(
               2 * this.keepaliveTimeMs,
               KEEPALIVE_MAX_TIME_MS
             );
+            logging.log(
+              LogVerbosity.ERROR,
+              `Connection to ${uriToString(this.channelTarget)} at ${this.subchannelAddressString} rejected by server because of excess pings. Increasing ping interval to ${this.keepaliveTimeMs} ms`
```

Comment:
Could technically be useful to have a different message if we reached the max, but, not a big deal.

Reply:
If this ever actually comes up, then I'll consider changing it.